src/test/java/com/amazonaws/neptune/auth/NeptuneSigV4SignerAbstractTest.java

```diff
@@ -229,8 +230,9 @@ public void attachSignatureHeaders() throws Exception {
         final String hostname = TEST_HOST_NAME;
         final String dateHeader = TEST_DATE_HEADER_VALUE;
         final String authHeader = TEST_AUTHORIZATION_HEADER_VALUE;
-
-        final NeptuneSigV4SignerBase.NeptuneSigV4Signature signature = new NeptuneSigV4SignerBase.NeptuneSigV4Signature(hostname, dateHeader, authHeader);
+        final String sessionToken = TEST_SESSION_TOKEN_VALUE;
```

Comment:
Can we add one test where the session token is not present?

Reply:
added test